src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,26 +47,165 @@ object ConductrPlugin extends AutoPlugin {
+                }
+              }
+            }
+          case _ =>
+            Def.taskDyn(Def.task[Option[InstallationData]](None))
+        }
+      }
+
+    def fold(a: Seq[InstallationData])(tasks: Seq[Def.Initialize[Task[Option[InstallationData]]]]): Def.Initialize[Task[Seq[InstallationData]]] =
+      tasks match {
+        case Nil =>
+          Def.task(a)
+        case x :: xs =>
+          Def.taskDyn {
+            x.value match {
+              case Some(v) => fold(a :+ v)(xs)
+              case None    => fold(a)(xs)
+            }
+          }
+      }
+
+    fold(List.empty)(bundleInstallationScriptsTasks)
+  }
+
+  private def generateInstallationScriptTask(): Def.Initialize[Task[File]] = Def.task {
+    val installDir = (target in LocalRootProject).value
+    val installationScript = installDir / "install.sh"
+    val installPath = installDir.toPath
+    val installationScriptContents = installationData.value.map {
+      case InstallationData(bundleName, bundle, bundleConfigFile) =>
+        val bundleIdEnv = bundleName.replaceAll("\\W", "_").toUpperCase + "_BUNDLE_ID"
+        val bundleArg = InstallationData.nameOrPath(
+          bundle match {
+            case Left(b)  => Left(b)
+            case Right(b) => Right(installPath.relativize(b))
+          }
+        )
+        val bundleConfigArg = bundleConfigFile.fold("")(installPath.relativize(_) + " ")
+        s"""
+           |echo "Deploying $bundleName..."
+           |$bundleIdEnv=$$(conduct load $bundleArg $bundleConfigArg--long-ids -q)
+           |conduct run $${$bundleIdEnv} --no-wait -q
+                 """.stripMargin
+    }
+    IO.write(
+      installationScript,
+      installationScriptContents.mkString(
+        s"""#!/usr/bin/env bash
+            |cd "$$( dirname "$${BASH_SOURCE[0]}" )"
+            |""".stripMargin,
+        "",
+        """
+          |echo 'Your system is deployed. Running "conduct info" to observe the cluster.'
+          |echo 'Type just "conduct" for help.'
+          |conduct info
+          |""".stripMargin
+      ),
+      Utf8
+    )
+    println("\nYour system's ConductR installation script: " + installationScript)
+    installationScript.setExecutable(true)
+    installationScript
+  }
+
+  private def installTask(): Def.Initialize[Task[Unit]] = Def.task {
+    withProcessHandling {
+      val nrOfContainers = "docker ps -q --filter='name=cond-'".lines_!.size
+      if (nrOfContainers > 0) {
+        println("Restarting ConductR to ensure a clean state...")
+        "docker exec cond-0 rm /opt/conductr/conf/seed-nodes".! // Allow the first node to form the new cluster
+        for (n <- 0 until nrOfContainers) s"docker restart -t 0 cond-$n".!
+      } else {
+        throw new IllegalStateException("Please first start the sandbox using 'sandbox run'.")
+      }
+    }(sys.error("There was a problem re/starting the sandbox."))
+
+    withProcessHandling {
+      waitForConductr(20.seconds)
```

Comment:
Should we make this duration a settingskey and therefore customizable by the user?